dask/dataframe/multi.py

```diff
@@ -1081,7 +1080,8 @@ def stack_partitions(dfs, divisions, join="outer", ignore_order=False, **kwargs)
                     join,
                     uniform,
                     filter_warning,
-                    ignore_order,
+                    False,
+                    kwargs,
```

Comment:
It looks like we are adding `False` here. Sorry if I missed this somewhere, but could you please remind me why that is?

Reply:
Ah I missed this one, added it for testing locally. Thanks for this catch John.